docs/reference/commandline/node_ps.md

```diff
@@ -93,8 +93,8 @@ The following filter matches tasks with the `usage` label regardless of its valu
 $ docker node ps -f "label=usage"
 
 NAME                               IMAGE        NODE            DESIRED STATE  CURRENT STATE
-redis.6.b465edgho06e318egmgjbqo4o  redis:3.0.6  swarm-manager1  Running        Running 10 minutes
-redis.7.bg8c07zzg87di2mufeq51a2qp  redis:3.0.6  swarm-manager1  Running        Running 9 minutes
+redis.6.b465edgho06e318egmgjbqo4o  redis:7.4.1  swarm-manager1  Running        Running 10 minutes
```

Comment:
Perhaps we should use `7.4` (or even `7`), to prevent these examples getting outdated. Generally, my recommendation would be to pin to either major, or major.minor, as patch releases should always be safe to update (and may contain security updates).

Reply:
I agree in principle, but I wanted to keep the patch version (and character count of this string) to avoid having to worry about whitespace alignment. If we change to 7 or 7.4, we need to fix the whitespace padding on the header row etc.

At some point I would like to replace the redis examples with something open source instead.

Reply:
Ah, right, that's fair.

I also left this comment before considering that one of the examples needs to show "updating", and I guess for that it makes more sense to update to a newer patch release (compared to "minor" release).